### PR TITLE
Ros2 idl compatibility

### PR DESCRIPTION
**Pull request by hendrik-skubch**
Companion PR:
https://github.com/rapyuta-robotics/alica-supplementary/pull/24

### CODE REVIEW

**Comment by dhananjaysathe** [comment]
+LGTM

**Review by dhananjaysathe — APPROVED** [review]
lgtm

File: alica_common_config/cmake/cflags.cmake [hunk 1]
```diff
@@ -4,7 +4,7 @@ set(C_WARNING "-Wbad-function-cast -Wstrict-prototypes")
 set(CPP_WARNING "")
 set(C_STD "-std=c11")
 set(CPP_STD "-std=c++11")
-set(OPTIMIZATION "-O2 -finline-functions -fomit-frame-pointer -funroll-loops -ffast-math")
+set(OPTIMIZATION "-O2 -finline-functions -fomit-frame-pointer -funroll-loops -ffast-math -ftree-vectorize")
```
Comment (dhananjaysathe) [inline alica_common_config/cmake/cflags.cmake:7]: why was this added as a part of this commit , what does it achieve ?

**Review by abolinsky — APPROVED** [review]
